runtimes/nodejs/nodejs.md

```diff
@@ -21,38 +21,33 @@ Node.js is a platform built on Chrome's JavaScript runtime for building fast, sc
 
 Refer to the page [Deploy an application on Clever Cloud](/doc/clever-cloud-overview/add-application/).
 
-## Necessary information
+## Requirements
 
 Be sure that:
 
 * you have pushed in <b>master branch</b>
 * you listen on <b>port 8080</b>
```

Comment:
it needs repeating that the app must listen on 0.0.0.0

(you can check rust doc which is up-to-date and states it)